src/mixins/audio_token_controls.js

```diff
@@ -91,28 +91,43 @@
       touchCornerSize: 64,
 
       cursorStyle: 'pointer',
+      mouseDownHandler: function (eventData, target) {
+        target.controls.playControl.heldDown = true;
+      },
```

Comment:
Shouldn't this be set to false in the mouseUpHandler?

Reply:
@msucharski, yes, I did it first. But in case we press down, move off the object, then release the button the event doesn't fire. That's why I set it to false in canvas mousUp event. Should we cover this case? Or how better to do?

Reply:
I the case of a user pressing down and moving off the button then mouseUp shouldn't trigger the button. I think that is how most buttons works.  One thing though is we need to reset the flag in that case.  Is there a event where we can check if the user has moved the mouse outside of the button?

Reply:
> I the case of a user pressing down and moving off the button then mouseUp shouldn't trigger the button. I think that is how most buttons works. One thing though is we need to reset the flag in that case. Is there a event where we can check if the user has moved the mouse outside of the button?

Created ticket weebug-3017 to find the way to detect if we are over a control for a fabric object, as @msucharski  advised